prog/dftb+/lib_dftbplus/initprogram.F90

```diff
@@ -4670,6 +4767,72 @@ subroutine getDenseDescCommon(orb, nAtom, t2Component, denseDesc)
   end subroutine getDenseDescCommon
 
 
+  !> Writes MBD-related info
+  subroutine writeMbdInfo(input)
+
+    !> MBD input parameters
+    type(TMbdInit), intent(in) :: input
+
+    character(80) :: tmpStr
+
+#:if WITH_MBD
+    write(stdOut, "(A)") ''
+    if (input%method == 'ts') then
+      write(stdOut, "(A)") "Using TS from SEDC module [Phys. Rev. B 80, 205414 (2009)]"
+      write(stdOut, "(A)") "PLEASE CITE: J. Chem. Phys. 144, 151101 (2016)"
+      select case (trim(input%vdw_params_kind))
+      case ('tssurf')
+        write(tmpStr, "(A)") "vdw-surf [Phys. Rev. Lett. 108, 146103 (2012)] "
+      case ('TSSURF')
+        write(tmpStr, "(A)") "vdw-surf [Phys. Rev. Lett. 108, 146103 (2012)] "
```

Comment:
This looks a bit redundant, how about this simplification?
```suggestion
      case ('tssurf', 'TSSURF')
        write(tmpStr, "(A)") "vdw-surf [Phys. Rev. Lett. 108, 146103 (2012)] "
```
Alternatively just make `input%vdw_params_kind` lowercase with `tolower`.